cmd/main.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/user"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/google/go-github/github"
+)
+
+var errorPlatformNotSupported error = errors.New("platform not supported, only supporting windows and drawin")
+
+func IsInstalled() (bool, error) {
+	platform := runtime.GOOS
+	execFilePath, err := getExecutableFile(platform)
+
+	if err != nil {
+		return false, err
+	}
+
+	return fileExists(execFilePath)
+}
+
+func Install() error {
```

Comment:
This should return the path where it was installed, as well as optionally accept a path.